tests/integration/study_data_blueprint/test_renewable.py

```diff
@@ -494,6 +494,30 @@ def test_lifecycle(
         assert other_cluster_name.upper() in description
         assert obj["exception"] == "DuplicateRenewableCluster"
 
+        # ===========================
+        #  VERSION 9.3
+        # ===========================
+
+        # Upgrade study to version 9.3
+        res = client.put(
+            f"/v1/studies/{internal_study_id}/upgrade",
+            headers={"Authorization": f"Bearer {user_access_token}"},
+            params={"target_version": 930},
+        )
+        res.raise_for_status()
+        task_id = res.json()
+        task = wait_task_completion(client, user_access_token, task_id)
+        assert task.status == TaskStatus.COMPLETED, task
+
+        # Checks creation with free `group`
+        res = client.post(
+            f"/v1/studies/{internal_study_id}/areas/{area_id}/clusters/renewable",
+            headers={"Authorization": f"Bearer {user_access_token}"},
+            json={"name": "free cluster", "group": "GroupFoo"},
+        )
+        assert res.status_code == 200, res.json()
+        assert res.json()["group"] == "groupfoo"
```

Comment:
Actually, this works, but when you GET the cluster from the backend, it's "other 1" (seen by testing with the front):

I think you forgot to adapt the serialization classes (`*FileData`).